### DISCUSSION TITLE
Custom layer at the bottom of the chart

### DISCUSSION THREAD

**Discussion by inomn** [plouc/nivo#2652] [Q&A] [answered]
Hi! Can a custom layer be added to the bottom of the chart (in my case it's a ResponsivePie -90' - 90') that will extend the SVG. I want to add numbers below the chart, see the screenshot 

![Screenshot 2024-09-25 at 21 45 14](https://github.com/user-attachments/assets/6b9fa886-1be8-4d4b-81fc-5415636e9fa8)

But as far as I can see - I can only render layers within SVG sizes, and I can't anyhow affect on the resulting SVG size with my custom layers, so I end up with a chart like this:

![Screenshot 2024-09-25 at 22 14 51](https://github.com/user-attachments/assets/3548929f-d8ea-46c2-a4a1-132fd5b2feab)

**Comment by plouc** [comment, marked as answer]
You could probably try to add some margin.
